modules/cco-ccoctl-install-creating-manifests.adoc

```diff
@@ -52,6 +69,23 @@ credentialsMode: Manual
 # ...
 ----
 
+ifdef::azure-workload-id[]
+. If you used the `ccoctl` utility to create a new Azure resource group instead of using an existing resource group, modify the `resourceGroupName` parameter in the `install-config.yaml` as shown:
++
+.Sample configuration file snippet
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: example.com
+# ...
+platform:
+  azure:
+    resourceGroupName: <azure_infra_name> <1>
+# ...
+----
+<1> This value must match the user-defined name for Azure resources that was specified with the `--name` argument of the `ccoctl azure create-all` command.
```

Comment:
The install resource group name can also be specified with the `--installation-resource-group-name` parameter, at which point it can be different from `--name`.

Reply:
For folks following along, we opted not to do this after discussing
https://redhat-internal.slack.com/archives/C04TMSTHUHK/p1700594776711419?thread_ts=1700489307.939559&cid=C04TMSTHUHK